script/Ciel/Day 8/08_01_CIEL08_1C.txt

```diff
@@ -292,32 +311,39 @@ Three minutes until homeroom starts.
 [sha:062e1d7bab217e7d38331ebe94a3e46c95eecb75]{
 -- Page 91, Offset 18479.
 -- 先輩とノエル先生が教室で顔を合わせるのはすごく胃が痛くなりそうだし、これはもうホントにしょうがない……。
-It'd be really nerve-wracking if Noel-sensei and Senpai were to meet inside the classroom, so I guess there isn't much of a choice...
+I feel like Senpai and Noel-sensei butting heads in the classroom would be enough to give me an ulcer,
+ so I guess there's nothing for it...
```

Comment:
I feel like the first clause made the sentence a bit more unclear than it should be.